raiden-dapp/src/views/Transfer.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="content-host">
+    <v-form v-model="valid" autocomplete="off">
+      <v-layout align-center justify-center row>
+        <v-flex xs10>
+          <address-input v-model="target" class="address-input"></address-input>
+        </v-flex>
+      </v-layout>
+
+      <v-layout align-center justify-center row>
+        <v-flex xs10>
+          <amount-input v-model="amount" :token="token"></amount-input>
+        </v-flex>
+      </v-layout>
+
+      <v-layout align-center justify-center row>
+        <div class="divider"></div>
+      </v-layout>
+
+      <v-layout align-center justify-center row class="information__wrapper">
+        <v-flex xs2 class="information">
+          <div class="information-label text-xs-left">Token</div>
+        </v-flex>
+        <v-flex xs8>
+          <div class="information-description text-xs-left">
+            <div>{{ token.symbol }} | {{ token.name }}</div>
+            {{ token.address }}
+          </div>
+        </v-flex>
+      </v-layout>
+
+      <v-layout align-center justify-center class="section">
+        <v-flex xs10 class="text-xs-center">
+          <v-btn
+            id="send"
+            class="text-capitalize confirm-button"
+            depressed
+            :disabled="!valid"
+            large
+            @click="transfer()"
+          >
+            Pay
+          </v-btn>
+        </v-flex>
+      </v-layout>
+    </v-form>
+    <progress-overlay
+      :display="loading"
+      :steps="steps"
+      :done-step="doneStep"
+      :done="done"
+    ></progress-overlay>
+    <error-screen
+      title="Something went wrong"
+      :description="error"
+      button-label="Dismiss"
+      @dismiss="error = ''"
+    ></error-screen>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import AddressInput from '@/components/AddressInput.vue';
+import AmountInput from '@/components/AmountInput.vue';
+import { StepDescription, Token, TokenPlaceholder } from '@/model/types';
+import { BalanceUtils } from '@/utils/balance-utils';
+import ProgressOverlay from '@/components/ProgressOverlay.vue';
+import ErrorScreen from '@/components/ErrorScreen.vue';
+
+@Component({
+  components: {
+    AddressInput,
+    AmountInput,
+    ProgressOverlay,
+    ErrorScreen
+  }
+})
+export default class Send extends Vue {
```

Comment:
Should the `Send` also be named `Transfer` for consistency?

Reply:
Yes, somehow I missed it.